javascripts/application.js

```diff
@@ -21,10 +21,12 @@ window.BeyondScheme = {
 // binding
 $(document).ready(function() {
 
-  $('.x-scroll-link').click(function(e) {
-    e.preventDefault();
-    var href = $(this).attr('href');
-    window.BeyondScheme.goToByScroll(href, 0, 0);
+  $('.x-scroll-contact').click(function(e) {
+    // scroll smoothly to contact section when on homepage
+    if (window.location.pathname == "/") {
```

Comment:
when you are not on homepage, for instance you are on /blog then we do nothing which means we just load link url from href attribute.